tests/system/python/data/testplugin.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Module for Test async mode plugin """
+import asyncio
+import copy
+import uuid
+import logging
+import datetime
+import async_ingest
+
+from foglamp.common import logger
+from foglamp.plugins.common import utils
+from foglamp.services.south import exceptions
+from threading import Thread
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+c_callback = None
+c_ingest_ref = None
+loop = None
+_task = None
+t = None
+
+_DEFAULT_CONFIG = {
+    'plugin': {
+        'description': 'Test Async Plugin',
+        'type': 'string',
+        'default': 'testplugin',
+        'readonly': 'true'
+    },
+    'assetPrefix': {
+        'description': 'Prefix of asset name',
+        'type': 'string',
+        'default': 'test-',
+        'order': '1',
+        'displayName': 'Asset Name Prefix'
+    },
+    'loudnessAssetName': {
+        'description': 'Loudness sensor asset name',
+        'type': 'string',
+        'default': 'loudness',
+        'order': '3',
+        'displayName': 'Loudness Sensor Asset Name'
+    }
+}
+
+_LOGGER = logger.setup(__name__, level=logging.INFO)
+
+
+def plugin_info():
+    """ Returns information about the plugin.
+    Args:
+    Returns:
+        dict: plugin information
+    Raises:
+    """
+    return {
+        'name': 'TEST Async Plugin',
+        'version': '2.0.0',
+        'mode': 'async',
+        'type': 'south',
+        'interface': '1.0',
+        'config': _DEFAULT_CONFIG
+    }
+
+
+def plugin_init(config):
+    """ Initialise the plugin.
+    Args:
+        config: JSON configuration document for the South plugin configuration category
+    Returns:
+        data: JSON object to be used in future calls to the plugin
+    Raises:
+    """
+    handle = copy.deepcopy(config)
+    return handle
+
+
+def plugin_start(handle):
+    """ Extracts data from the sensor and returns it in a JSON document as a Python dict.
+    Available for async mode only.
+    Args:
+        handle: handle returned by the plugin initialisation call
+    Returns:
+        returns a sensor reading in a JSON document, as a Python dict, if it is available
+        None - If no reading is available
+    Raises:
+        TimeoutError
+    """
+    global _task, loop, t
+    loop = asyncio.new_event_loop()
+    _task = asyncio.ensure_future(_start_aiotest(handle), loop=loop)
+
+    def run():
+        global loop
+        loop.run_forever()
+
+    t = Thread(target=run)
+    t.start()
+
+async def _start_aiotest(handle):
+
+    ts_lst = []
+    ts_lst.append(str(datetime.datetime.now(datetime.timezone.utc).astimezone()))
```

Comment:
May be 
`from datetime import datetime as dt, timezone as tz, timedelta`
or just 
`from datetime import datetime, timezone, timedelta`